gp_emulator/gpu/_gpu_predict.c

```diff
@@ -0,0 +1,157 @@
+#include "gpu_predict.h"
+#include<errno.h>
+
```

Comment:
If this is based on template code taken from an external source, then please always put a link into the original on which it was based.